### PR TITLE
[msvc] Include IR for fixes MSVC compiling error.

### PR DESCRIPTION
**Pull request by lygstate**

### CODE REVIEW

**Comment by nadavrot** [comment]
Why is this needed? Can you please post the error message that you are seeing in MSCV? 

Also, please provide a clear commit message.

File: examples/cifar10.cpp [hunk 1]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 #include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/IR/IR.h"
```
Comment (rdzhabarov) [inline examples/cifar10.cpp]: In cifar10 I do not see anything which would require IR.h. Could you please paste corresponding compilation errors?

**Comment by rdzhabarov** [comment]
@lygstate are you planning to continue working on this and related PRs?

**Comment by lygstate** [comment]
I am busy now, continue when I have the time, anyone wanna help would be fine

**Comment by rdzhabarov** [comment]
Closing, more details: https://github.com/pytorch/glow/issues/1144#issuecomment-423013661

**Pull request closed without merging**